joomla/joomla-cms
# Improvements in the usability of the debug console

Pull Request for Improvement.
#### Summary of Changes

This PR adds a fixed position and ability to drag the debug console.

Now the page is always the same but with a debug console that is fixed in a position. No need to scroll down to the bottom of the page to check the debug results.
#### Testing Instructions
1. Use latest staging.
2. Enable the debug console.
3. Check the debug console is always on the bottom of the page (you have to scroll down always).
4. Apply patch.
5. Check the debug console is always fixed on top right of the screen and opens on clear.
6. Check you can drag the console to another location if you need (with the drag icon).
7. On a long page, scroll down and see if the debug console is always there.

![image](https://cloud.githubusercontent.com/assets/9630530/14531588/e079bc4a-0255-11e6-9580-69a04baa3af7.png)
#### Observations

Please note the js files added in this PR are the `jquery ui` `draggable` events.

Improvements suggestions are welcomed.
